.github/config.yml

```
@@ -0,0 +1,29 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted on first-time issues
+newIssueWelcomeComment: >
+  🎉 **Welcome to the project, @{{user.login}}!** 🎉  
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)